### ISSUE TITLE
It's not possible to navigate the Mini Cart contents with the keyboard if it has inner blocks

### ISSUE THREAD

**Issue by Aljullu** [woocommerce/woocommerce-gutenberg-products-block#6575]
## Describe the bug

When the Mini Cart contents have inner blocks added by the user, it stops being accessible with the keyboard and it's not possible to focus product inner blocks.

## To reproduce

1. In the site editor, add the Mini Cart block to your site header.
2. Edit the Mini Cart template part and add an image in the inner contents.
3. In the frontend, add some products to the Cart.
4. With the keyboard, open the Mini Cart.
5. With the <kbd>Tab</kbd> key try to navigate to the Mini Cart products.
6. Notice you can't focus the Mini Cart products inner elements.

## Expected behavior

I should be able to focus the Mini Cart items title, quantity selector, delete item button, etc.

## Screenshots

https://user-images.githubusercontent.com/3616980/173779723-967873b9-1f3f-4ea3-9a9f-8aa721277264.mp4
